pegasus/test/test_minecraft_certificates.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './test_helper'
+require_relative '../../dashboard/lib/script_constants'
+require 'minitest/autorun'
+
+class MinecraftCertificatesTest < Minitest::Test
+  def setup
+    @minecraft_certificates = ['mee', 'mee_empathy', 'mee_timecraft', 'mee_estate'] << ScriptConstants::CATEGORIES[:minecraft]
```

Comment:
Should 'mee', 'mee_empathy', 'mee_timecraft', 'mee_estate' be added to the minecraft script constants or are they not categories?

Reply:
From my understanding, these are separate from the other script constants because they are `Minecraft: Education Edition`. They are checked for here: https://github.com/code-dot-org/code-dot-org/blob/dfe028e8c5d3cdfd97e528c64ef9cb9005e4469e/dashboard/legacy/certificate_image.rb#L224.
Im not quite sure exactly why they are separated from the others.

Reply:
Oh gotcha, thank you!